flink-core/src/test/java/org/apache/flink/util/TestLogger.java

```diff
@@ -37,23 +37,26 @@
 
 		@Override
 		public void starting(Description description) {
-			log.info("================================================================================");
-			log.info("Test {} is running.", description);
-			log.info("--------------------------------------------------------------------------------");
+			log.info("================================================================================"
```

Comment:
I would a new line here before the line as well. Then we have everything together with the same indentation.